client/notifications/src/panel/templates/empty-message.jsx

```diff
@@ -2,7 +2,6 @@ import React, { Component } from 'react';
 
 import { bumpStat } from '../rest-client/bump-stat';
 
-// from $title-offset in boot/sizes.scss
```

Comment:
`$title-offset: 38px;` from `sizes.scss` wasn't used anywhere so this seemed redundant.

Reply:
probably the `$wpnc__title-bar-height`

do you have any experience matching JS and CSS values?

Reply:
sass/css variables + css modules? :trollface: 

I typically try avoiding these and when I cannot, I try to read the value from DOM element, and when I cannot rely on that, I prefer to have a frontend test that ensures things stay in sync as intended.

...often I don't have luxury of none of those so I just do what has been done here and cross my fingers someone doesn't break it. 😆  I'll add the comment back with `$wpnc__title-bar-height` :thumbsup: